.github/workflows/tests.yml

```diff
@@ -193,7 +193,7 @@ jobs:
       fail-fast: false # remove once tests aren't flaky
       matrix:
         project: ['chromium', 'webkit', 'firefox']
-        shard: [1/2, 2/2]
+        shard: [1/4, 2/4, 3/4, 4/4]
```

Comment:
time out is set to 60 minutes, does it makes sense to reduce it now that we use more shards?

Reply:
If we ever see it get stuck and hit the timeout we can reduce it to avoid wasting that time, but so far I haven't seen it happen before, I don't want to reduce it and possibly introduce more flakiness